larrecodnn/NuGraph/Tools/CMakeLists.txt

```diff
@@ -1,6 +1,15 @@
 cet_enable_asserts()
 
-cet_make_library(LIBRARY_NAME NuGraphBaseTools INTERFACE SOURCE LoaderToolBase.h DecoderToolBase.h)
+cet_make_library(LIBRARY_NAME NuGraphBaseTools
+                 INTERFACE SOURCE LoaderToolBase.h DecoderToolBase.h
+                 LIBRARIES INTERFACE
+                   lardataobj::RecoBase
+                   lardataobj::AnalysisBase
```

Comment:
What header include causes the need for lardataobj::AnalysisBase?

Reply:
ah thanks for catching this! it's used in one of the .cc files, not in the headers. Let me remove it